src/components/auth/LoginWithAuthServerPage.tsx

```diff
@@ -17,6 +18,7 @@ enum View {
 
 export const LoginWithAuthServerPage = () => {
   const [view, setView] = useState<View>(View.LOADING)
+  const [isLoading, setIsLoading] = useState(false)
   const [disabled, setDisabled] = useState(false)
```

Comment:
should we remove this? or is it being used somewhere else?

Reply:
It's still being used while waiting for the user to sign the identity in the browser.